File: cfgs/java_cfgs_foo/FIND_UNIT_DIGIT_X_RAISED_POWER_Y_1.java
Nodes
1;special;;;
2;statement;x = foo_1 % 10;14;
3;conditional;foo_1 != 0;15;
4;statement;y = (foo_1 % 4) + 4;15;
5;statement;return ((int) (fun_1(foo_1, foo_2))) % 10;16;
6;exit;;;
Edges
0;2;
1;3;
2;4;5;
3;5;
4;6;
5;

File: cfgs/pythoncfgs/FIND_UNIT_DIGIT_X_RAISED_POWER_Y_1.py
Nodes
1;statement;foo_1 = foo_2 % 10;10;
2;conditional;foo_1 != 0;11;
3;statement;return fun_2(fun_2(foo_1, foo_2)) % 10;13;
4;statement;foo_1 = foo_2 % 4 + 4;12;
5;exit;;
Edges
0;2;
1;3;4;
2;5;
3;3;
4;
